Rhino.Licensing/LicenseGenerator.cs

```diff
@@ -125,15 +178,15 @@ private static XmlDocument CreateDocument(Guid id, string name, DateTime expirat
             var idAttr = doc.CreateAttribute("id");
             license.Attributes.Append(idAttr);
             idAttr.Value = id.ToString();
-        
+
             var expirDateAttr = doc.CreateAttribute("expiration");
             license.Attributes.Append(expirDateAttr);
             expirDateAttr.Value = expirationDate.ToString("yyyy-MM-ddTHH:mm:ss.fffffff", CultureInfo.InvariantCulture);
 
             var licenseAttr = doc.CreateAttribute("type");
             license.Attributes.Append(licenseAttr);
             licenseAttr.Value = licenseType.ToString();
-            
+

```

Comment:
Can you extract this trim change to its own maint commit along with the other styling only changes?

Reply:
Done!